### PR TITLE
port doc/make.py to py3k

### PR DESCRIPTION
**Pull request by sandrotosi**
we applied this patch in the last update to Debian, since the we are trying to remove python 2

### CODE REVIEW

File: doc/make.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def latex():
 
         os.chdir('../..')
     else:
-        print 'latex build has not been tested on windows'
+        print('latex build has not been tested on windows')
```
Comment (WeatherGod) [inline doc/make.py]: Need to add a `from __future__ import print_function` at the top so that this will still work in the python 2 series.

**Comment by WeatherGod** [comment]
Keep in mind that there is no plan to drop python 2 and upgrade to py3k only because support for basemap will end in Jan. 2020.

**Comment by sandrotosi** [comment]
I wasnt suggest to drop py2 at all, but you can build the documentation on py3k while still having the actual module support 2 and 3

**Comment by WeatherGod** [comment]
There is no compelling reason to drop py2 doc building if the module itself is still written in python 2. Because there is no plan for moving on to py3k for basemap, dropping py2 doc-building seems entirely unnecessary.

**Comment by WeatherGod** [comment]
Note, in case it wasn't clear, I am all for getting doc-building working on py3k. It is trivial to support both py2 and py3 for doc-building in this case.

File: doc/make.py (hunk 1 above)
Comment (sandrotosi) [reply doc/make.py]: done

**Comment by WeatherGod** [comment]
Thanks for the patch. Note to self, I need to update the travis-ci file to unbreak the testing

**Pull request merged**